src/Vulkan/Silk.NET.Vulkan/Vk.cs

```diff
@@ -362,5 +362,12 @@ public void PurgeExtensionCache()
                 _cachedDeviceExtensionsLock.ExitWriteLock();
             }
         }
+
+        private IVTable VulkanCreateVTable()
+        {
+            var ret = CreateVTable();
+            ret.Initialize(Context, CoreGetSlotCount());
+            return ret;
```

Comment:
Isn't this just what CreateDefaultContext does?

Reply:
Nope, the VTable is initialized in the NativeApiContainer ctor.